worker/backup_ee.go

```diff
@@ -1,3 +1,4 @@
+//go:build !oss
```

Comment:
I'm not familiar with this'go:build' type tag. Intentional?

Reply:
sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)